### DISCUSSION TITLE
[Bug]: What is anonymousId?

### DISCUSSION THREAD

**Discussion by victorkennedys** [oddbit/flutter_facebook_app_events#277] [General]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Hi! I am wondering what anonymousId is, i.e. what is returned in : ` await facebookAppEvents.getAnonymousId();`

Is it the same as attribution id? More info here: https://developers.facebook.com/docs/marketing-api/app-event-api/#testing

Thank you for a great plugin!

### Expected Behavior

.

### Steps To Reproduce

.

### Platform

iOS

### Flutter Doctor

```shell
.
```

**Comment by DennisAlund** [comment]
Use discussions or stackoverflow for discussions. Thank you.

**Comment by JuanFcoRomero** [comment]
The Facebook Anonymous ID is a required ID for the facebook integration for iOS devices to be able to to track these events properly. It's essential to track events accurately and make sure our 3rd party integration with Facebook ads runs smoothly.
